src/test/resources/features/account.feature

```diff
@@ -8,12 +8,19 @@ Feature: Requests for Account endpoint
   @CreateAccount
   Scenario Outline: Create a Account
     Given I build the pathParams and body to request
```

Comment:
Would be better described as "I set the pathParams and body for a POST request"

Reply:
Good suggestion. I will change it.